capability/capability_linux.go

```diff
@@ -117,6 +117,11 @@ func newPid(pid int) (c Capabilities, retErr error) {
 	return
 }
 
+func ignorableError(err error) bool {
+	// Ignore EINVAL as not supported on kernels before 4.3
```

Comment:
See, the original code has two different comments about why we're ignoring EINVAL:
1. For `PR_CAPBSET_DROP`, it says "Ignore EINVAL since the capability may not be supported in this system."
2. For `PR_CAPSET_AMBIENT`, it says "Ignore EINVAL as not supported on kernels before 4.3."

With this change, you remove the first comment, moved the second one into this function, and thus it is now applicable to both cases, which is incorrect.